contracts/modules/oracle/ArbitrumOracle.sol

```diff
@@ -3,7 +3,7 @@
 
 // import  'forge-std/console.sol';
```

Comment:
remove. we should never have leftover `console.log` even if commented out in  contract code